src/test/ui/lint/lint-unused-mut-variables.rs

```diff
@@ -4,9 +4,18 @@
 #![allow(unused_variables)]
 #![allow(dead_code)]
 #![deny(unused_mut)]
+#![feature(param_attrs)]
 
+//fn baz(#[allow(unused_mut)] mut b: i32) {}
```

Comment:
Why are these commented out?

Reply:
I forgot to include a `FIXME` because it is currently failing.
Still trying to figure out how to make this lint work